src/api/api.ts

```diff
@@ -2,14 +2,13 @@ import axios, { Method } from 'axios';
 
 // Base Axios instance
 const api = axios.create({
-    baseURL: 'https://mfsl.loca.lt',
+    baseURL: 'http://localhost:8080',
```

Comment:
Should we make this an env file so it can be overriden on the deployed version

Reply:
Will address this at the end of the project